### PR TITLE
Add Amusiz flatpak manifest

### PR DESCRIPTION
**Pull request by mirkobrombin**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: pm.mirko.Amusiz.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+id: pm.mirko.Amusiz
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: '40'
+command: amusiz
+finish-args:
+- "--allow=multiarch"
```
Comment (barthalion) [inline pm.mirko.Amusiz.yml:7]: What is it needed for?

File: pm.mirko.Amusiz.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- "--share=network"
+- "--share=ipc"
+- "--socket=fallback-x11"
+- "--socket=pulseaudio"
+- "--socket=wayland"
+- "--system-talk-name=org.freedesktop.GeoClue2"
```
Comment (barthalion) [inline pm.mirko.Amusiz.yml:13]: Same question. (As geolocation in music app seems odd without context.)

**Comment by barthalion** [comment]
bot, build pm.mirko.Amusiz

File: pm.mirko.Amusiz.yml (hunk 2 above)
Comment (mirkobrombin) [reply pm.mirko.Amusiz.yml:13]: I used the manifest from another application of mine and forgot to remove it

File: pm.mirko.Amusiz.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+- "--socket=wayland"
+- "--env=IS_FLATPAK=1"
+modules:
+- name: amusiz
+  builddir: true
+  buildsystem: meson
+  sources:
+  - type: git
+    url: https://github.com/mirkobrombin/Amusiz
+    release: 1.3.2
```
Comment (hfiguiere) [inline pm.mirko.Amusiz.yml:20]: Didn't you mean `tag`? `release` isn't a valid property.

**Comment by hfiguiere** [comment]
also not sure what's the course of action, but the Pull request is against the wrong branch. It should be against `new-pr`, not `master`

File: pm.mirko.Amusiz.yml (hunk 3 above)
Comment (mirkobrombin) [reply pm.mirko.Amusiz.yml:20]: ok I definitely pushed the wrong manifest

**Pull request closed without merging**